CONTRIBUTING.md

```diff
@@ -24,71 +24,38 @@ In order to build **[nimi-python](https://github.com/ni/nimi-python)**, you must
+    - If you're on Windows 10 (not using WSL):
         - [Install mingw (msys-base)](https://osdn.net/projects/mingw/).
-        - Add <mingw Install Path>\msys\1.0\bin to Windows path.
+        - Add ``mingw install path``\msys\1.0\bin to Windows path.
 
     - If you're on macOS
         - Install [Xcode](https://itunes.apple.com/us/app/xcode/id497799835?mt=12)
         - Install command line developer tools
 
-* [PyPI](https://pip.pypa.io/en/latest/installing/)
+* [PyPI](https://pip.pypa.io/en/latest/installation/)
 
-        sudo apt-get install python3-pip
-        sudo pip3 install pip --upgrade
+        sudo apt-get install python3-pip (Linux)
+        sudo pip3 install pip --upgrade (Windows)
 
 * Additional Python Modules (install using [PyPI](https://pypi.python.org/pypi))
 
-        sudo pip install tox tox-globinterpreter --upgrade
-
-* Configure tox interpreter search - only required on Windows when not using WSL
-    - Copy <path to 64 bit python3>\python.exe <path to 64 bit python3>\python3.exe
-        - This can be any supported version of python 3
-    - cd to GitHub repo location
-    - <path to python 2>\python -m tox --scan <path 1>\python.exe <path 2>\python3.exe
-        - You must use Python 2 for this step
-        - path 1 
-            - Can not use spaces, instead you can use wildcards
-            - Ex: c:/Program\*/Python\*/python.exe
-        - path 2
-            - Should be the path to the 64 bit python 3 where python3.exe exists
-        - Ex: `"c:\Program Files (x86)\Python27\python" -m tox --scan c:/Program*/Python*/python.exe c:/Program*/Python*/python3.exe`
-        - This should list out each version of python installed plus one more for python3
-
-                interpreters:
-                python2.7 c:\Program Files\Python27\python.exe
-                python2.7 c:\Program Files (x86)\Python27\python.exe
-                python3.6 c:\Program Files\Python36\python.exe
-                python3.6 c:\Program Files (x86)\Python36\python.exe
-                python3 c:\Program Files\Python36\python3.exe
-
-        - Ex: `"c:\Python27\python" -m tox --scan c:/Python*/python.exe c:/Python*/python3.exe`
-        - This should list out each version of python installed plus one more for python3
-
-                interpreters:
-                python2.7 c:\Python27\python.exe
-                python2.7 c:\Python27-32\python.exe
-                python3.6 c:\Python36\python.exe
-                python3.6 c:\Python36-32\python.exe
-                python3 c:\Python36\python3.exe
-
-In order to run **[nimi-python](https://github.com/ni/nimi-python)** System Tests:
-
-* Install corresponding driver runtimes.
+        sudo pip install pytest tox --upgrade
+
+In order to run the **[nimi-python](https://github.com/ni/nimi-python)** system tests:
+
+* Install the corresponding driver runtimes.
     * Download the latest installers for NI-DMM, NI-SCOPE, NI-DCPower, NI-SWITCH, NI-FGEN, NI Switch Executive
```

Comment:
`Download the latest installers for NI-DCPower, NI-Digital Pattern Driver, NI-DMM, NI-FGEN, NI-SCOPE, NI-SWITCH, and NI Switch Executive from [ni.com](http://www.ni.com/downloads/ni-drivers/)`

(Added NI-Digital and alphabetized the list)